kernel/src/main/java/org/kframework/parser/inner/generator/RuleGrammarGenerator.java

```diff
@@ -129,9 +132,10 @@ public Module getRuleGrammar(Module mod) {
     public Module getConfigGrammar(Module mod) {
         // import CONFIG-CELLS in order to parse cells specific to configurations
         Module newM = new Module( mod.name() + "-" + CONFIG_CELLS
-                                , Set(mod, baseK.getModule(K).get(), baseK.getModule(CONFIG_CELLS).get(), baseK.getModule(DEFAULT_LAYOUT).get())
-                                , Set()
-                                , Att()
+                                , (scala.collection.Set<Module>) mod.publicImports().$bar(Set(baseK.getModule(K).get(), baseK.getModule(CONFIG_CELLS).get(), baseK.getModule(DEFAULT_LAYOUT).get()))
+                                , mod.privateImports()
+                                , mod.localSentences()
+                                , mod.att()
                                 );
```

Comment:
This pattern than `getConfigGrammar` uses seems like it could be extracted into a single function `getGrammar`, which takes as input `Module mod, String cellName`, then `getConfigGrammar(Module mod) => getGramma(mod, mod.name() + "-" + CONFIG_CELLS)`, for instance. That would deduplicate `getRuleGrammar` and `getConfigGrammar`